app/javascript/packages/address-search/types.d.ts

```diff
@@ -1,4 +1,5 @@
 import type { RegisterFieldCallback } from '@18f/identity-form-steps';
+import { ReactNode } from 'react';
```

Comment:
Can this be the following instead?

```suggestion
import type { ReactNode } from 'react';
```